src/wp-includes/class-walker-nav-menu.php

```diff
@@ -170,9 +187,30 @@ public function start_el( &$output, $data_object, $depth = 0, $args = null, $cur
 		 * @param int      $depth        Depth of menu item. Used for padding.
 		 */
 		$id = apply_filters( 'nav_menu_item_id', 'menu-item-' . $menu_item->ID, $menu_item, $args, $depth );
-		$id = $id ? ' id="' . esc_attr( $id ) . '"' : '';
 
-		$output .= $indent . '<li' . $id . $class_names . '>';
+		$li_atts 		  = array();
```

Comment:
```suggestion
		$li_atts          = array();
```
Should fix this PHPCS failure.